plugins/Jobs.cpp

```diff
@@ -25,6 +25,13 @@ const set<string, STableComp> BedrockPlugin_Jobs::supportedRequestVerbs = {
     "RequeueJobs",
 };
 
+bool BedrockJobsCommand::canEscalateImmediately(SQLiteCommand& baseCommand) {
+    // This is a set of commands that we will escalate to leader without waiting. It's not intended to be complete but
+    // to solve the biggest issues we have with slow escalation times (i.e., this is usually a problem for `FinishJob`).
+    static const set<string> commands = {"CreateJob", "CreateJobs", "FinishJob"};
```

Comment:
We are moving more traffic to master with this, because `CreateJob` and `CreateJobs` can finish on peek...
I am fine with this change for now, but we should at least remove those from the list later.